cdec/installation-manager-cli-bundle/src/main/java/com/codenvy/im/cli/command/AddNodeCommand.java

```diff
@@ -78,8 +79,13 @@ protected void doExecuteCommand() throws Exception {
     protected void validateCodenvyConfig() throws IOException {
         if (isCodenvy4Installed()) {
             Config config = configManager.loadInstalledCodenvyConfig();
-            String property = config.getProperties().get(Config.MACHINE_EXTRA_HOSTS);
 
+            // check on adding default node
+            if (NodeConfigHelper.isDefaultNode(config, dns)) {
```

Comment:
It is better to avoid invoking `validateCodenvyConfig` method in case of DefaultNode instead of skipping validation inside the method.